### ISSUE TITLE
Missing heated bed designing file

### ISSUE THREAD

**Issue by kqkq** [Ultimaker/Ultimaker-Original-Plus#1]
The part `1155 Print Table Heated Bed` is an electronic part. But there's only a 3D model available. How can I get an EDA file (e.g. Eagle or Altium PCB file) ?

**Comment by daid** [comment]
The part is the same as in the Ultimaker2, so the files should be there as well.

**Comment by kqkq** [comment]
@daid 

No, the EDA file is not there either. I open an issue (https://github.com/Ultimaker/Ultimaker2/issues/10) EIGHT months ago and no one reply.

**Comment by harmawoldhuis** [comment]
I'm sorry it took so long. I uploaded the correct files a few weeks ago.

2015-04-09 12:36 GMT+02:00 KUANG Qi notifications@github.com:
